### DISCUSSION TITLE
Order trash items by time of deletion not by overall date

### DISCUSSION THREAD

**Discussion by C8opmBM** [tutao/tutanota#7735] [Bugs]
Hey,
So this happened to me today, I think this shouldn't be the natural way TRASH folder should handle messages.

So I deleted by mistake a message, but I couldn't remember which message was.
Normally, you'd open trash and the last message (the one on top) should be the one deleted. 
In tuta, the trash items are sorted by date, so if you didn't clean your trash for a while, it can be impossible to find it.

I suggest to arrange the items by the time they were trashed.

Thanks!

**Comment by igorpupkinable** [comment]
Not sure this is a bug. Sounds like a minor feature request potentially not worth the effort as old email do not sit in the Trash for more than 30 days.

**Comment by C8opmBM** [comment]
I consider this a bug, since the natural way of handling thrash is the one I depicted above. 
Depending on your inbox size, and due to the fact that the message list cannot be customized to be thinner, it can get really bad in finding specific messages in the trash folder.
This is the natural way of all emails being trashed, I don't see a reason why on tuta needs to be different.
I appreaciate the work that's being done, this is a small request to be added to the backlog.
Thanks!
